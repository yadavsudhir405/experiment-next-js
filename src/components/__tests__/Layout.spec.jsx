import React from 'react'
import {render} from "@testing-library/react";

import { withLayout }from "../index";

jest.mock('../Clock', () => () => <div>This is dummy Clock</div>);
jest.mock('../Navbar', () => () => <div>This is dummy Navbar</div>);

describe('WithLayout', () => {
   it('should match the snapshot', () => {
       const dummyComponent = () => <div>This is dummy Component</div>;
       const DummyComponent = withLayout(dummyComponent);

       const {container} = render(<DummyComponent dummyProps="dummy" />);

       expect(container).toMatchSnapshot();
   })
});
