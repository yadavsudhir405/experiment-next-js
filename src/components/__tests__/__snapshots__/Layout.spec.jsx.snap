// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithLayout should match the snapshot 1`] = `
<div>
  <div>
    <div>
      This is dummy Navbar
    </div>
    <div>
      This is dummy Clock
    </div>
    <div>
      This is dummy Component
    </div>
  </div>
</div>
`;
